Documentation/Storage-Configuration/ceph-teardown.md

```diff
@@ -108,8 +108,12 @@ DISK="/dev/sdX"
 # Zap the disk to a fresh, usable state (zap-all is important, b/c MBR has to be clean)
 sgdisk --zap-all $DISK
 
-# Wipe a large portion of the beginning of the disk to remove more LVM metadata that may be present
-dd if=/dev/zero of="$DISK" bs=1M count=100 oflag=direct,dsync
+# Wipe portions of the disk to remove more LVM metadata that may be present
```

Comment:
@puskunalis Thanks for the PR.  It would be great if this can be converted into a small shell script.

Reply:
I agree with @sp98 : a short  script will be more convenient.

And instead of executing the various commands verbatim in a row, The script could loop over the different offset, adding the offset as a variable in the loop's body which would essentially consist of one parametrised `dd`command.

Reply:
How about if we just maintain the current doc as it is, rather than create a separate shell script? Perhaps a future PR could create a separate script.

Reply:
@travisn wrote:
> How about if we just maintain the current doc as it is, rather than create a separate shell script? Perhaps a future PR could create a separate script.

I think we were not discussing about a separate script but about a script embedded in the doc for copying and pasting.

Reply:
We chatted about this in huddle. Rook used to keep a more detailed version of this doc, but it got too cumbersome to maintain. This helpful but not fully prescriptive doc has done well for Rook for a while, so let's not add complexity of making this a script we would need to e2e test. As-is, I think this looks fine, with a couple questions I have below